Makefile

```diff
@@ -61,7 +62,8 @@ tilt-ci:
 .PHONY: integration-test
 integration-test:
 	KUBECONFIG='./.kubeconfig-dev' minikube update-context
-	go test ./... -tags=integration_test -count=1 -race
+	mkdir -p $(TEST_COVERAGE_DIR)
+	go test ./... -tags=integration_test -count=1 -race -coverprofile=$(TEST_COVERAGE_DIR)/integration-coverage.out -covermode=atomic
```

Comment:
I'm excited you want to add test coverage for integration tests, but [coverage profile support for integration tests requires extra work](https://go.dev/doc/build-cover)

Reply:
Since this is a time-consuming task, let's skip it for now